app/services/cache/HistoricDocumentRequestSearchCache.scala

```diff
@@ -78,20 +78,10 @@ class HistoricDocumentRequestSearchCache @Inject()(appConfig: AppConfig,
   }
 
   def retrieveDocumentsForCurrentEori(currentEori: String): Future[Seq[HistoricDocumentRequestSearch]] =
-    collection.find(equal(currentEoriFieldKey, currentEori)).toFuture() recover {
```

Comment:
What's our thinking removing the exception handling and logging from the retrieves?

Reply:
It has been removed to propagate the exception to the calling method and then, those exceptions are being handled (and exception message is being logged) in the controller (Using Try) in order to produce the required InternalServerError (500).